<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsCompatibilityOptionsPage</class>
 <widget class="QWidget" name="TagsCompatibilityOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>705</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="tag_compatibility">
     <property name="title">
      <string>ID3</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QGroupBox" name="id3v2_version">
        <property name="title">
         <string>ID3v2 Version</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QRadioButton" name="write_id3v24">
           <property name="text">
            <string>2.4</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="write_id3v23">
           <property name="text">
            <string>2.3</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="id3v2_text_encoding">
        <property name="title">
         <string>ID3v2 text encoding</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QRadioButton" name="enc_utf8">
           <property name="text">
            <string>UTF-8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="enc_utf16">
           <property name="text">
            <string>UTF-16</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="enc_iso88591">
           <property name="text">
            <string>ISO-8859-1</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hbox_id3v23_join_with">
        <item>
         <widget class="QLabel" name="label_id3v23_join_with">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Join multiple ID3v2.3 tags with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="id3v23_join_with">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default is '/' to maintain compatibility with previous Picard releases.&lt;/p&gt;&lt;p&gt;New alternatives are ';_' or '_/_' or type your own. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string notr="true">/</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">; </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true"> / </string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="itunes_compatible_grouping">
        <property name="text">
         <string>Save iTunes compatible grouping and work</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="write_id3v1">
        <property name="text">
         <string>Also include ID3v1 tags in the files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>274</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>write_id3v24</tabstop>
  <tabstop>write_id3v23</tabstop>
  <tabstop>enc_utf8</tabstop>
  <tabstop>enc_utf16</tabstop>
  <tabstop>enc_iso88591</tabstop>
  <tabstop>id3v23_join_with</tabstop>
  <tabstop>itunes_compatible_grouping</tabstop>
  <tabstop>write_id3v1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
